### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Ahmed-Benj**
Hello TSE,

We hope you are doing well.

We have finished the project : POSITIONING AND FLOATING ELEMENTS.

Coul you please review our project and indicate if there are any issues that need to be fixed ?

Kind regards,

### CODE REVIEW

**Comment by christianotieno** [comment]
## Pull Request under review by Christian Otieno (TSE)

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @karthykarthick @Ahmed-Benj  :wave:
-
You have done a good job! :tada:
We, however, do have some few changes to carry out before we proceed (see review comment).
##
- Kindly take note on the differences in the two pages below and work on the suggested changes. The views are based on a desktop (large screen view).

## Top Page and main Section 

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/karthykarthick/New-York-Times/development/index.html)|
|:---:|:---:|
|main-page|main-page-clone|
- From the comparison, there are some noticeable differences. Kindly, let's improve on the main image's caption's positioning. The contents below the image and it's caption including the social icon's section needs to be centrally placed. Kindly work on that.

## Inflation of Theory Section
|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/karthykarthick/New-York-Times/development/index.html)|
|:---:|:---:|
|inflation-of-theory|inflation-of-theory-clone|

- Let's improve on the theory of inflation section. the contents are not well placed as they should. Let's work on that, please.

## More in Space Section
|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/karthykarthick/New-York-Times/development/index.html)|
|:---:|:---:|
|more-in-space|more-in-space-clone|

- More work needs to be done in the `more in space section` 

## Footer Section
|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/karthykarthick/New-York-Times/development/index.html)|
|:---:|:---:|
|footer|footer-clone|

- More work needs to be done in the footer section as well. Let's improve on the styling.
##

- Also, make sure that the footer should have a fixed position (it sticks to the bottom part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)).


- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by christianotieno** [comment]
> Hello TSE,
> 
> We hope you are doing well.
> 
> We have finished the project : POSITIONING AND FLOATING ELEMENTS.
> 
> Coul you please review our project and indicate if there are any issues that need to be fixed ?
> 
> Kind regards,

## Good job with the summary. We can always improve on it though by highlighting what we have carried out in the project's PR to guide the reviewer on what has been achieved so far. 
- We just have to pay attention to what's being communicated and adhere to these guidelines of having a meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
- Also, be sure to take care of grammar and spelling mistake in your PR and README file as well.

**Comment by karthykarthick** [comment]
final changes made and corrected our errors

**Comment by karthykarthick** [comment]
final changes made based on previous review

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @Ahmed-Benj  and @karthykarthick! 👋 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- As was pointed out to you by the previous reviewer, there are still some differences in the layout between your clone and the original page:

- [ ] The author's name, the date, and the social media icons are not properly aligned below the image. Please compare your clone with the original page in the screenshots below and fix this.
Your clone 👇 
Screen Shot 2020-06-23 at 7 12 01 PM
The original page 👇 
Screen Shot 2020-06-23 at 7 12 31 PM

- [ ] Your clone is missing the `Editor's Picks` section. Please compare your clone with the original page and add this missing section. 
The original page 👇 
Screen Shot 2020-06-23 at 7 14 01 PM
Your clone 👇 
Screen Shot 2020-06-23 at 7 14 31 PM

- [ ] The text above the `Theory of Inflation` section should only span the width of the images.
Your clone 👇 
Screen Shot 2020-06-23 at 7 16 52 PM
The original page 👇 
Screen Shot 2020-06-23 at 7 17 09 PM

- [ ] In your clone, the `Theory of inflation` section is missing a heading. Please compare your clone with the original page and fix this. 
The original page 👇 
Screen Shot 2020-06-23 at 7 17 59 PM
Your clone 👇 
Screen Shot 2020-06-23 at 7 18 34 PM

- [ ] When scrolling, your navigation bar is overlapping with some of the content on your screen. Please fix this.
Your clone above the `theory of inflation`👇 
Screen Shot 2020-06-23 at 7 20 07 PM
Your clone above `most popular`👇 
Screen Shot 2020-06-23 at 7 20 46 PM

- [ ] The `Read 615 comments` button and the social media icons right below the article are not properly aligned. Please compare your clone with the original page and fix this.
Your clone 👇 
Screen Shot 2020-06-23 at 7 24 13 PM
The original page 👇 


- [ ] More work needs to be done on your typography. There are several instances where the section headings are not as large as they should be or are not as bold as they are in the original page.
Your clone 👇 
Screen Shot 2020-06-23 at 7 28 55 PM
The original page 👇 
Screen Shot 2020-06-23 at 7 29 58 PM
Your clone 👇 
Screen Shot 2020-06-23 at 7 30 23 PM
The original page 👇 
Screen Shot 2020-06-23 at 7 30 46 PM
Your clone 👇 
Screen Shot 2020-06-23 at 7 31 28 PM
The original page 👇 
Screen Shot 2020-06-23 at 7 31 45 PM
Your clone 👇 
Screen Shot 2020-06-23 at 7 32 18 PM
The original page 👇 
Screen Shot 2020-06-23 at 7 32 34 PM

- [ ] The footer is missing some of the content that is present on the original page and is not styled in the same way. This was pointed out to you in your first review. Please compare yours with the original and fix this. 
Your clone 👇 
Screen Shot 2020-06-23 at 7 35 13 PM
The original page 👇 
Screen Shot 2020-06-23 at 7 35 41 PM

****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+}
+
+i.fa {
+  display: inline-block;
+  border-radius: 60px;
+  padding: 0.5em 0.6em;
+}
+
+.caption {
+  /* margin: 20px 0 0 114px; */
```
Comment (BrittanyBlake) [inline style.css]: Please remove the commented code. It makes your file look untidy. If you need to review a previous version of your code, use git history.

**Review by trekab — CHANGES_REQUESTED** [review]
## Changes Required! :recycle: 
Hi @Ahmed-Benj and @karthykarthick! I commend your efforts on this project so far. :+1: There are some requirements to implement in order to complete this project, though. These include:
- [ ] Your navigation bar isn't filling the entire width of the viewport.
  Screnshot

- [ ] The NY Times logo should be a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))
- [ ] The main content should generally be centered. Yours appears to be more to the left.
  Screenshot
- [ ] Your `More in Space and Astronomy` & `Most Popular` section:
  - Need to have some right and left margin.
  - `Most Popular` heading is misplaced.
  - `Editor's Picks` heading is missing.
  - You have unnecessary white space between both sections.
Screenshot
- [ ] You should have a footer that has a fixed position (it sticks to the bottom part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Comment by karthykarthick** [comment]
fixed issues based on the previous review

**Comment by karthykarthick** [comment]
there are some issues with the resolution we don't know  how to fix it

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Great job guys so far :clap: :clap:, your project looks good but in order to make it get approved you need to make some changes.
### status: require changes :recycle: 

### :warning:  I have seen that you didn't make the required changes by the previous TSE, please next time you have a code review make all the mentioned changes cause it takes us a lot of effort to review your code.

- Fix the header as mentioned by the previous TSE
- Fix the main content as described by the previous TSE "The main content should generally be centered. Yours appears to be more to the left."
- Fix the More in Space and Astronomy & Most Popular sections as described by the previous TSE, here is how it should look:
image

### Please make the required changes before you request another code  review
## Happy Coding.
_If you are stuck at any point or you have questions don't hesitate to contact me on slack_

**Comment by karthykarthick** [comment]
previous errors fixed
my resolution
in my resolution it is perfect

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @karthykarthick and @Ahmed-Benj, congratulations on fixing the comments by the previous TSE :+1: However, I found a bug we need to fix. Kindly check the images below for a quick fix :slightly_smiling_face: 
 
f1
f2
f3
f4
f5
fc

Kindly address these comments and submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](linkedin.com/in/daniel-larbi-addo/)

**Comment by karthykarthick** [comment]
issues fixed based on the previous review and in my resolution, it was working perfectly 

resolution1

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes
## sorry I can't approve this project until you fix the required changes, here is how it looks in my computer and I guess it's the same for all the previous TSE's:
image

## contact when you see this comment so we can range a solution or if you were stuck I can help you out.
### Happy Coding

**Comment by karthykarthick** [comment]
I
my screen
 take a screenshot from git hub pages in my system it is perfect 
previous TSE are viewing different resolution I think it causes a problem and also I attach a screenshot from GitHub pages for my resolution

**Review by trekab — CHANGES_REQUESTED** [review]
## Changes Required! :recycle: 
Hi @team, well done! You are almost there. There are a few fixes to make to complete this project. Check out the screenshot below. I have pointed out areas that should be fixed. Try to compare your clone to the original in the marked areas and adjust accordingly.
screenshot

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Comment by karthykarthick** [comment]
changes made based on the previous review

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change required :recycle: 

### Hi everyone

### Good job with the project, it gets better and better, unfortunately some elements of the page are messed up and have to be fixed, I will attach below those sections:
- This section is overlapping over the text:
Screenshot from 2020-06-25 11-49-09
- The video should be center with the paragraph
Screenshot from 2020-06-25 11-50-29
- also this picture should centered
Screenshot from 2020-06-25 11-51-37
- The text is overlapping over the images:
Screenshot from 2020-06-25 11-52-31
 ### Once done with the change, ask for another code request.

**Comment by karthykarthick** [comment]
fixed previous issues

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project @karthykarthick. :fireworks: Keep it up.
##
***All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***
- Congratulations on the splendid job once more!
- You can now merge these changes.
- I wish you all the very best in your next tasks 👇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**